src/redisgears_python.c

```diff
@@ -1088,6 +1092,22 @@ static int RedisGearsPy_Execut(RedisModuleCtx *ctx, RedisModuleString **argv, in
     return REDISMODULE_OK;
 }
 
+static void onDoneResultsOnly(ExecutionPlan* ep, void* privateData){
+    RedisModuleBlockedClient *bc = privateData;
+    RedisModuleCtx *rctx = RedisModule_GetThreadSafeContext(bc);
+    Command_ReturnResults(ep, rctx);
+    RedisModule_UnblockClient(bc, NULL);
+    RedisGears_DropExecution(ep);
+    RedisModule_FreeThreadSafeContext(rctx);
+}
+
+int RedisGearsPy_ExecuteReturnResultsOnly(RedisModuleCtx *ctx, RedisModuleString **argv, int argc){
+    doneFunction = onDoneResultsOnly;
```

Comment:
thought maybe to give this function the callback as an arg and implement the onDoneResultsOnly on the command.c, what do you think?

Reply:
That was my original thinking and understanding from you. However. I found it difficult for technical reasons (or own lack of knowledge) - lets slack on this.

Reply:
Resolved via 635f722